provider-mdr/cnv-workload/appset/vm-appset-1/vm-appset-1.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: vm-appset-1
+  namespace: openshift-gitops
+spec:
+  generators:
+    - clusterDecisionResource:
+        configMapRef: acm-placement
+        labelSelector:
+          matchLabels:
+            cluster.open-cluster-management.io/placement: vm-appset-1-placement
+        requeueAfterSeconds: 180
+  template:
+    metadata:
+      name: vm-appset-1-{{name}}
+      labels:
+        velero.io/exclude-from-backup: "true"
+    spec:
+      project: default
+      source:
+        repoURL: https://github.com/red-hat-storage/ocs-workloads
```

Comment:
This needs to be changed accordingly. We need to reference the workload under `provider-mdr`